src/components/modal/examples/ModalCompositeConnectedExamples.tsx

```diff
@@ -116,6 +153,14 @@ export class ModalCompositeConnectedExamples extends React.Component<IModalExamp
                             closeCallback={() => alert('The modal has closed.')}
                         />
                     </div>
+
+                    <div className='mt3'>
+                        <label className='form-control-label'>Modal With a Prevent Navigation</label>
+                        <div className='form-group'>
+                            <button className='btn' onClick={() => this.props.openModal('modal-composite-2')}>Open Modal</button>
+                            <ComponentWithPreventNavigationHOC />
+                        </div>
+                    </div>

```

Comment:
I don't see this example in the demo, maybe it has not finished building yet?

Reply:
It looks like an issue with the build 🤔